src/v2/components/Search/index.jsx

```diff
@@ -44,26 +44,40 @@ const Search = () => {
       return;
     }
 
-    const filteredValidators = filter(v => {
+    const filteredValidators = reduce((acc, v) => {
```

Comment:
This code was slightly complicated/non-obvious to me at first
with the repetition. Do you have any way to factor that out &
make the code a little easier to understand?

const condition = (x) => { A || B || C };
const results = filter(condition)(validators);

Reply:
@sunnygleason it was like that before actually, look at #370 issue. these changes were made to solve it and i have to use reducers here and check for condition after each one. This way i can understand resource type.

This code will be gone after we move to search api method.

Reply:
@manuel-calavera ok I'll show you what I mean in a subsequent PR.